plugin/evm/wrapped_database.go

```diff
@@ -0,0 +1,66 @@
+// (c) 2019-2020, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package evm
+
+import (
+	"errors"
+
+	"github.com/ava-labs/avalanchego/database"
+	"github.com/ethereum/go-ethereum/ethdb"
+)
+
+var (
+	_ ethdb.KeyValueStore = &WrappedDatabase{}
+
+	ErrSnapshotNotSupported = errors.New("snapshot is not supported")
+)
+
+// WrappedDatabase implements ethdb.Database
```

Comment:
Is this still the case? Should we add type asserts like `_ ethdb.Database = &WrappedDatabase{}`?